packages/front-end/components/MetricExperiments/MetricExperiments.tsx

```diff
@@ -188,7 +188,10 @@ function MetricExperimentResultTab({
         </td>
         <td>
           <div className="my-1">
-            <ExperimentStatusIndicator status={e.status} />
+            {/* FIXME: How can we get the archived status? */}
```

Comment:
Looks like we'll need to update `getExperimentsUsingMetrics` ([here](https://github.com/growthbook/growthbook/blob/4da03b9d283d5c6e2034bad3fc54778f9d6595b4/packages/back-end/src/models/ExperimentModel.ts#L661)) - Probably makes sense to add an optional prop for this that defaults to not including archived experiments.